agentassist/src/app/pages/coaching/trigger/speech-analysis/speech-analysis.component.html

```diff
@@ -48,7 +48,7 @@
         <ul class="dropdown_menu-wrapper" ngbDropdownMenu>
             <div class="seconds-input">
                 <div class="kr-sg-input-text">
-                    <input type="number" step="0.01" placeholder="Enter Seconds here" class="input-text" (keyup.enter)="onEnterTime($event.target.value);deadairtimerdropdown.close()">
+                    <input type="number" step="0.01" placeholder="Enter Seconds here" class="input-text" (keyup.enter)="onEnterTime(this.showTimer);deadairtimerdropdown.close()" [(ngModel)]="showTimer">
```

Comment:
@saradadh-Kore why do you need to pass the variable as it is already available in .ts file